src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -1246,6 +1246,54 @@ Copyright (c) .NET Foundation. All rights reserved.
                       Overwrite="true" />
   </Target>
 
+  <Target Name="GenerateBundledMSBuildProps" DependsOnTargets="SetupBundledComponents">
+    <PropertyGroup>
+      <MinimumMSBuildVersionFile>$(RedistLayoutPath)sdk/$(Version)/minimumMSBuildVersion</MinimumMSBuildVersionFile>
+      <BundledMSBuildPropsFileName>Microsoft.NETCoreSdk.BundledMSBuildInformation.props</BundledMSBuildPropsFileName>
+      <BundledMSBuildVersion>$(MSBuildVersion)</BundledMSBuildVersion>
+    </PropertyGroup>
+
+    <Error Text="No MSBuild version file found under '$(RedistLayoutPath)sdk/$(Version)'" Condition="!Exists('$(MinimumMSBuildVersionFile)')" />
+    
+    <ReadLinesFromFile File="$(MinimumMSBuildVersionFile)">
+      <Output TaskParameter="Lines" PropertyName="MinimumMSBuildVersion"/>
+    </ReadLinesFromFile>
+
+    <PropertyGroup>
+      <_BundledMSBuildVersionMajorMinor>$([System.Version]::Parse('$(BundledMSBuildVersion)').ToString(2))</_BundledMSBuildVersionMajorMinor>
+    </PropertyGroup>
+
+    <PropertyGroup>
+      <BundledMSBuildPropsFileContent>
+<![CDATA[
+<!--
+***********************************************************************************************
+$(BundledMSBuildPropsFileName)
```

Comment:
Is this resolved by the WriteLinesToFile task?

Reply:
No, it's resolved by evaluating the contents of the property - this is pretty copy/paste from the bundled props and CLI tools files generated in this targets file already.

Anything with a $ in this prop gets evaluated in installer, anything with a %24 is evaluated in user projects.